content/compose/compose-file/05-services.md

```diff
@@ -1500,7 +1491,6 @@ section of the Docker run reference page.
 
 `runtime` specifies which runtime to use for the service’s containers.
 
-The value of `runtime` is specific to the implementation.
 For example, `runtime` can be the name of [an implementation of OCI Runtime Spec](https://github.com/opencontainers/runtime-spec/blob/master/implementations.md), such as "runc".
```

Comment:
We could call out that `runc` is the default, and to use a different runtime you need to install the shims and/or configure the daemon as described here: https://docs.docker.com/engine/alternative-runtimes/